apps/coscrad-frontend/src/components/resources/vocabulary-lists/vocabulary-list-detail.full-view.presenter.tsx

```diff
@@ -1,21 +1,95 @@
-import { IDetailQueryResult, IVocabularyListViewModel } from '@coscrad/api-interfaces';
+import {
```

Comment:
We need to test the filtering behaviour in the detail integration test. Very important!